esphome/wizard.py

```diff
@@ -367,10 +367,7 @@ def wizard(path):
     )
     safe_print()
     safe_print("Next steps:")
-    safe_print(
-        '  > Check your Home Assistant "integrations" screen. If all goes well, you '
-        "should see your ESP being discovered automatically."
-    )
-    safe_print("  > Then follow the rest of the getting started guide:")
+    safe_print("  > Follow the rest of the getting started guide:")
     safe_print("  > https://esphome.io/guides/getting_started_command_line.html")
```

Comment:
Let's link to the section where we are:
```suggestion
    safe_print("  > https://esphome.io/guides/getting_started_command_line.html#adding-some-features")
```